packages/sirius-web/frontend/sirius-web-application/src/views/project-browser/create-projects-area/ProjectTemplatesModal.tsx

```diff
@@ -78,14 +81,16 @@ export const ProjectTemplatesModal = ({ onClose }: ProjectTemplatesModalProps) =
     count: null,
   });
 
+  const { canCreate, canUpload } = useContext<ProjectsCapabilitiesContextValue>(ProjectsCapabilitiesContext);
```

Comment:
Why should we perform this check again? Isn't is done server side?

Reply:
I used an enum instead of a boolean and I deleted the "other" cards